### PR TITLE
Add support for SHA1 pdb checksums and fix MSBuild logging

### PR DESCRIPTION
**Pull request by AArnott**

### CODE REVIEW

File: src/GitLink/Linker.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright file="Linker.cs" company="CatenaLogic">
-//   Copyright (c) 2014 - 2014 CatenaLogic. All rights reserved.
+// <copyright file="Linker.cs" company="Andrew Arnott">
```
Comment (GeertvanHorrik) [inline src/GitLink/Linker.cs]: Not sure why this change is required? Not that I mind much, but we are considering moving to the .net foundation anyway. Just wondering if this is intended.

File: src/GitLinkTask/GitLink.cs [hunk 2]
```diff
@@ -30,7 +26,7 @@ public class GitLink : Task
 
         public override bool Execute()
         {
-            LogManager.GetCurrentClassLogger().LogMessage += this.LinkProject_LogMessage;
+            LogManager.AddListener(new MSBuildListener(this.Log));
```
Comment (GeertvanHorrik) [inline src/GitLinkTask/GitLink.cs]: Please remove the "this.", we don't use it anywhere

File: src/GitLinkTask/GitLink.cs [hunk 3]
```diff
@@ -43,25 +39,35 @@ public override bool Execute()
             return success && !this.Log.HasLoggedErrors;
         }
 
-        private void LinkProject_LogMessage(object sender, LogMessageEventArgs e)
+        private class MSBuildListener : LogListenerBase
         {
-            switch (e.LogEvent)
+            private readonly TaskLoggingHelper log;
```
Comment (GeertvanHorrik) [inline src/GitLinkTask/GitLink.cs]: _log

**Comment by GeertvanHorrik** [comment]
Changes look good. Wondering why the change for copyright is required, since parts would officially still be copyrighted differently. We could consider dropping the headers as a whole, but that's something for a new PR.

File: src/GitLinkTask/GitLink.cs (hunk 2 above)
Comment (AArnott) [inline src/GitLinkTask/GitLink.cs]: Removed everywhere in this file (except the places it was required).

File: src/GitLinkTask/GitLink.cs (hunk 3 above)
Comment (AArnott) [inline src/GitLinkTask/GitLink.cs]: done. crying just a little more.
